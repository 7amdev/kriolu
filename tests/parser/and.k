verdadi e 1 e "text";
1 e 2 e falsu e 3 e 4;