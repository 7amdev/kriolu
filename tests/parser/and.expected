<expression statement>
  (verdadi e (1.0 e 'text'))
<expression statement>
  (1.0 e (2.0 e (falsu e (3.0 e 4.0))))